src/QueryCategorizer.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Scanner;
+
+public class QueryCategorizer {
+    private final QueryKeeper queryKeeper = new QueryKeeper();
+
+    private void separateBySign(String[] query) {
+        for (String queryIterator : query) {
+            if (queryIterator.contains("+")) {
+                queryKeeper.getPlusContain().add(queryIterator.substring(1));
+            } else if (queryIterator.contains("-")) {
+                queryKeeper.getMinusContain().add(queryIterator.substring(1));
+            } else {
+                queryKeeper.getWithOutSign().add(queryIterator);
```

Comment:
inke hey ye property az class dige ro call mikonin va behesh add mikonid kar khobi nis
shoma biayn 3ta set tarif konin poresh konin va baad az for bian set konin be field hay class query keeper